ci-operator/step-registry/ibmcloud/deprovision/vpc/ibmcloud-deprovision-vpc-commands.sh

```diff
@@ -105,8 +105,14 @@ echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpc_name
 
 "${IBMCLOUD_CLI}" target -g ${resource_group} -r ${region}
 
-echo "DEBUG" "Removing the vpc ${vpc_name}"
+echo "DEBUG" "Removing the vpc ${vpc_name} ..."
 delete_vpc "${vpc_name}"
 
+echo "DEBUG" "Removing the resource reclamations ..."
+set +o errexit
+${IBMCLOUD_CLI} resource reclamations |  awk '(NR>4) {print $1}' | xargs -n1 ibmcloud resource reclamation-delete -f
```

Comment:
This commands will be always executed, even the resources are not created by this install workflow?